agir/events/components/eventPage/EventReportCard.js

```diff
@@ -0,0 +1,50 @@
+import { DateTime } from "luxon";
+import React from "react";
+import styled from "styled-components";
+import Collapsible from "../../../front/components/genericComponents/Collapsible";
+import Button from "../../../front/components/genericComponents/Button";
+import Card from "../../../front/components/genericComponents/Card";
+
+const StyledButton = styled(Button)`
+  height: 48px;
+  font-size: 16px;
+  font-weight: 500;
+  border-radius: 0.5rem;
```

Comment:
On a une variable maintenant pour la nouvelle valeur de border-radius :
```suggestion
  border-radius: ${style.borderRadius};
```